shell/platform/fuchsia/flutter/platform_view.cc

```diff
@@ -578,12 +578,18 @@ void PlatformView::HandlePlatformMessage(
   if (!message) {
     return;
   }
-  auto found = platform_message_handlers_.find(message->channel());
+  const std::string channel = message->channel();
+  auto found = platform_message_handlers_.find(channel);
   if (found == platform_message_handlers_.end()) {
-    FML_LOG(ERROR)
-        << "Platform view received message on channel '" << message->channel()
-        << "' with no registered handler. And empty response will be "
-           "generated. Please implement the native message handler.";
+    const bool already_errored = unregistered_channels_.count(channel);
```

Comment:
why do we need this additional check? if we're not in `unregistered_channels_` then we've never logged before. This currently will log if we get a message on e.g. channelA, then channelB, channelC, etc won't log (as count() will be > 0).

Reply:
This is `count(channel)`, as in a count to check if the channel is in the `set`.

Reply:
oh I misread. sorry!